### PR TITLE
Release 1.1.0

### PR DESCRIPTION
**Pull request by carpanz**
In this PR we include:

- Template and nf-core/modules update to the latest version
- Addition of Krona and Circos plot as example to documentation
- Addition of local module samtools/viewsingle. When extracting reads, we noticed that also some _both_unmapped_ reads were collected in _single_unmapped_ reads due to a conflict between flags option. In order to resolve this issue, we create a local module where we execute the extraction in two step (i.e. we combine two command with a pipe) instead of running the module twice, which would otherwise be inefficient.

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/hgtseq/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/hgtseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by lescai — APPROVED** [review]
Reviewed in dev already. 
Code looks good to me, all checks are passed.
Important to release the bug fix. This is also a chance to sync the pipeline to latest version of modules and software as well as to strengthen documentation.

File: modules/local/samtools/viewsingle/main.nf [hunk 1]
```diff
@@ -0,0 +1,70 @@
+process SAMTOOLS_VIEW_SINGLE {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "bioconda::samtools=1.17"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/samtools:1.15.1--h1170115_0' :
```
Comment (FriederikeHanssen) [inline modules/local/samtools/viewsingle/main.nf:7]: Sam tools version not updated. Looks like you too this module from nf-core and modified it. You could also install the module as normal and then apply "patch". That way version changes are automatically installed

File: nextflow.config [hunk 2]
```diff
@@ -208,12 +232,13 @@ dag {
...
-    author          = 'Simone Carpanzano, Francesco Lescai'
+    author          = """Simone Carpanzano, Francesco Lescai"""
     homePage        = 'https://github.com/nf-core/hgtseq'
-    description     = 'A pipeline to investigate horizontal gene transfer from NGS data'
+    description     = """A pipeline to investigate horizontal gene transfer from NGS data"""
     mainScript      = 'main.nf'
-    nextflowVersion = '!>=21.10.3'
-    version         = '1.0.0'
+    nextflowVersion = '!>=22.10.1'
+    version = '1.1.0'
```
Comment (FriederikeHanssen) [inline nextflow.config:240]: The indent is off

**Review by FriederikeHanssen — CHANGES_REQUESTED** [review]
Almost there 🚀

File: nextflow.config (hunk 2 above)
Comment (carpanz) [reply nextflow.config]: done!

File: modules/local/samtools/viewsingle/main.nf (hunk 1 above)
Comment (carpanz) [reply modules/local/samtools/viewsingle/main.nf]: Resolved! Discussed offline on Slack.

**Review by FriederikeHanssen — APPROVED** [review]
🚀  great work

**Pull request merged**